just init ERR_MODULE_NOT_FOUND

### Describe the bug

`just init` fails with `ERR_MODULE_NOT_FOUND` because `justfile` line 21 runs `node packages/tools/src/index.ts sync-remote` directly. Node.js cannot resolve `.js` extension imports (e.g. `import('./sync-remote-deps.js')`) to `.ts` files without a TypeScript loader.

The correct entry point is `packages/tools/src/bin.js`, which registers `@oxc-node/core/register` before importing `index.ts`.

### Reproduction

Clone the repo and run: ``` just init ```

### Steps to reproduce

1. `git clone` the repo
2. Run `just init`
3. Observe `ERR_MODULE_NOT_FOUND` for `sync-remote-deps.js`

### System Info

```shell
Node.js v22.18.0
just 1.48.1
```

### Used Package Manager

pnpm

### Logs

```shell
node packages/tools/src/index.ts sync-remote
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/.../packages/tools/src/sync-remote-deps.js' imported from /Users/.../packages/tools/src/index.ts
```

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/voidzero-dev/vite-plus/blob/main/CONTRIBUTING.md).
- [x] Check that there isn't [already an issue](https://github.com/voidzero-dev/vite-plus/issues) for the same bug.
- [x] Confirm this is a Vite+ issue and not an upstream issue (Vite, Vitest, tsdown, Rolldown, or Oxc).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).

--- a/justfile
+++ b/justfile
@@ -18,7 +18,7 @@ _clean_dist:
 
 init: _clean_dist
   cargo binstall watchexec-cli cargo-insta typos-cli cargo-shear dprint taplo-cli -y
-  node packages/tools/src/index.ts sync-remote
+  node packages/tools/src/bin.js sync-remote
   pnpm install
   pnpm -C docs install
 

